### PR TITLE
DC-389: freeze plugin & tsaas client version for release 31

### PR DESCRIPTION
**Pull request by freddy61025**
https://issues.opennms.org/browse/DC-389

- change ver. to 1.0.0 (plugin & tsaas client)
- packages & features name opennms-cloud-plugin > opennms-plugin-cloud (align with other plugin)
- fix fakeroot problem in circle-ci

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -23,7 +23,7 @@
         <karaf.version>4.3.6</karaf.version>
         <log4j.version>2.18.0</log4j.version>
         <mockito.version>2.18.0</mockito.version>
-        <opennms.api.version>1.0.1</opennms.api.version>
+        <opennms.api.version>1.1.0</opennms.api.version>
```
Comment (patrick-schweizer) [inline pom.xml:26]: I wonder if we are ok to have here a fixed version? Can we install the plugin on a higher Api version? I have never tried that...

Comment (freddy61025) [reply pom.xml:26]: this is expected OIA version for H31. I do test it manually yesterday looks fine.

File: it-test/src/test/java/org/opennms/plugins/cloud/ittest/EndToEndIt.java [hunk 2]
```diff
@@ -107,11 +107,11 @@ public void canInstallAndStartPlugin() {
     private void installAndStartPluginInOpenNms() {
 
         // install plugin for core
-        opennmsShell.runCommand("kar:install --no-start mvn:org.opennms.plugins.cloud.assembly/org.opennms.plugins.cloud.assembly.kar/1.0.0-SNAPSHOT/kar");
-        opennmsShell.runCommand("feature:install opennms-cloud-plugin-core");
+        opennmsShell.runCommand("kar:install --no-start mvn:org.opennms.plugins.cloud.assembly/org.opennms.plugins.cloud.assembly.kar/1.0.0/kar");
```
[the remark this answers is not in the document]
Comment (freddy61025) [reply it-test/src/test/java/org/opennms/plugins/cloud/ittest/EndToEndIt.java:110]: issue created https://issues.opennms.org/browse/DC-404

File: it-test/src/test/java/org/opennms/plugins/cloud/ittest/EndToEndIt.java [hunk 3]
```diff
@@ -125,11 +125,11 @@ private void installAndStartPluginInOpenNms() {
     private void installAndStartPluginInSentinel() {
 
         // install plugin for core
-        sentinelShell.runCommand("kar:install --no-start mvn:org.opennms.plugins.cloud.assembly/org.opennms.plugins.cloud.assembly.kar/1.0.0-SNAPSHOT/kar");
-        sentinelShell.runCommand("feature:install opennms-cloud-plugin-sentinel");
+        sentinelShell.runCommand("kar:install --no-start mvn:org.opennms.plugins.cloud.assembly/org.opennms.plugins.cloud.assembly.kar/1.0.0/kar");
```
[the remark this answers is not in the document]
Comment (freddy61025) [reply it-test/src/test/java/org/opennms/plugins/cloud/ittest/EndToEndIt.java:128]: same as above

File: it-test/src/test/resources/docker-compose.yaml [hunk 4]
```diff
@@ -28,10 +28,10 @@ services:
     image: "opennms/deploy-base:jre-2.0.6.b165"
     hostname: cloudMock
     entrypoint: /bin/bash
-    command: ["-c", "java -cp /usr/share/opennms/.m2/repository/org/opennms/plugins/cloud/it-test/1.0.0-SNAPSHOT/it-test-1.0.0-SNAPSHOT-jar-with-dependencies.jar org.opennms.plugins.cloud.ittest.MockCloudMain"]
+    command: ["-c", "java -cp /usr/share/opennms/.m2/repository/org/opennms/plugins/cloud/it-test/1.0.0/it-test-1.0.0-jar-with-dependencies.jar org.opennms.plugins.cloud.ittest.MockCloudMain"]
```
[the remark this answers is not in the document]
Comment (freddy61025) [reply it-test/src/test/resources/docker-compose.yaml:31]: same as above

**Review by patrick-schweizer — APPROVED** [review]

**Pull request merged**